### ISSUE TITLE
Title view in navbar not visible on targets below 11.0

### ISSUE THREAD

**Issue by m3g0byt3** [m3g0byt3/Podo#5]
iOS 11.x:
![screenshot 2017-12-13 23 39 07](https://user-images.githubusercontent.com/21194149/33961392-f29cdd88-e05e-11e7-83e5-b00d2df817e9.png)

iOS 10.x:
![screenshot 2017-12-13 23 39 19](https://user-images.githubusercontent.com/21194149/33961397-f695b1bc-e05e-11e7-8670-c0da47e9b06f.png)

**Comment by m3g0byt3** [comment]
Probably an autolayout/SnapKit issue, Xcode autolayout warnings on iOS 9.x/10.x targets:

```
2017-12-13 23:43:24.482108+0300 Podo[10883:232107] [LayoutConstraints] Unable to simultaneously satisfy constraints.
	Probably at least one of the constraints in the following list is one you don't want. 
	Try this: 
		(1) look at each constraint and try to figure out which you don't expect; 
		(2) find the code that added the unwanted constraint or constraints and fix it. 
	(Note: If you're seeing NSAutoresizingMaskLayoutConstraints that you don't understand, refer to the documentation for the UIView property translatesAutoresizingMaskIntoConstraints) 
(
    "<NSAutoresizingMaskLayoutConstraint:0x60000008c300 h=--& v=--& Podo.TitleView:0x7fb236c17010.width == 0   (active)>",
    "<SnapKit.LayoutConstraint:0x6100000b2180@NibSettableView.swift#30 UIView:0x7fb236d0afd0.left == Podo.TitleView:0x7fb236c17010.left>",
    "<SnapKit.LayoutConstraint:0x6100000b2420@NibSettableView.swift#30 UIView:0x7fb236d0afd0.right == Podo.TitleView:0x7fb236c17010.right>",
    "<SnapKit.LayoutConstraint:0x6000000b8540@TitleView.swift#18 UIImageView:0x7fb236d0c650.left == UIView:0x7fb236d0afd0.left + 5.0>",
    "<SnapKit.LayoutConstraint:0x6000000b8660@TitleView.swift#18 UIImageView:0x7fb236d0c650.right == UIView:0x7fb236d0afd0.right - 5.0>"
)

Will attempt to recover by breaking constraint 
<SnapKit.LayoutConstraint:0x6000000b8660@TitleView.swift#18 UIImageView:0x7fb236d0c650.right == UIView:0x7fb236d0afd0.right - 5.0>

Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKit/UIView.h> may also be helpful.
2017-12-13 23:43:24.483425+0300 Podo[10883:232107] [LayoutConstraints] Unable to simultaneously satisfy constraints.
	Probably at least one of the constraints in the following list is one you don't want. 
	Try this: 
		(1) look at each constraint and try to figure out which you don't expect; 
		(2) find the code that added the unwanted constraint or constraints and fix it. 
	(Note: If you're seeing NSAutoresizingMaskLayoutConstraints that you don't understand, refer to the documentation for the UIView property translatesAutoresizingMaskIntoConstraints) 
(
    "<NSAutoresizingMaskLayoutConstraint:0x60000008c3a0 h=--& v=--& Podo.TitleView:0x7fb236c17010.height == 0   (active)>",
    "<SnapKit.LayoutConstraint:0x6100000b2060@NibSettableView.swift#30 UIView:0x7fb236d0afd0.top == Podo.TitleView:0x7fb236c17010.top>",
    "<SnapKit.LayoutConstraint:0x6100000b23c0@NibSettableView.swift#30 UIView:0x7fb236d0afd0.bottom == Podo.TitleView:0x7fb236c17010.bottom>",
    "<SnapKit.LayoutConstraint:0x6000000b8600@TitleView.swift#18 UIImageView:0x7fb236d0c650.top == UIView:0x7fb236d0afd0.top + 5.0>",
    "<SnapKit.LayoutConstraint:0x6000000b86c0@TitleView.swift#18 UIImageView:0x7fb236d0c650.bottom == UIView:0x7fb236d0afd0.bottom - 5.0>"
)

Will attempt to recover by breaking constraint 
<SnapKit.LayoutConstraint:0x6000000b86c0@TitleView.swift#18 UIImageView:0x7fb236d0c650.bottom == UIView:0x7fb236d0afd0.bottom - 5.0>

Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKit/UIView.h> may also be helpful.
`
```

**Comment by m3g0byt3** [comment]
Fixed in [3d76a1c](https://github.com/m3g0byt3/Podo/commit/3d76a1c7f336383e703c172fe70258d1aeeba325).
